### PR TITLE
New game Star Trek Online 

### PR DESCRIPTION
**Pull request by ImperatorS79**
I know the game is working under wine 2.12-staging. However, since I can't test my scripts with current POL5, I'm not sure the script is working ...

### CODE REVIEW

**Comment by plata** [comment]
Blocked until PlayOnLinux/POL-POM-5#957 is solved.

**Comment by ImperatorS79** [comment]
Since the issue has been solved, I can now confirm that the script and the program are working flawlessly

File: Games/Star Trek Online/Steam/script.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+new SteamScript()
+    .name("Star Trek Online")
+    .editor("Cryptic Studios")
+    .author("ImperatorS79")
+    .appId(9900)
+    .wineVersion("2.12")
+    .wineDistribution("staging")
+    
+.go();
```
Comment (plata) [inline Games/Star Trek Online/Steam/script.js]: Just to be consistent with the other scripts: could you remove the first empty line (also in script.json) and indent the `.go()`?

Comment (ImperatorS79) [inline Games/Star Trek Online/Steam/script.js]: it's done

**Pull request merged**